### ISSUE TITLE
 revisit how we define/use `processIds` for these status messages across the repo.

### ISSUE THREAD

**Issue by BrianWhitneyAI** [AllenInstitute/biofile-finder#352]
outside of this PR's scope, but noting that we may want to revisit how we define/use `processIds` for these status messages across the repo since it's currently somewhat ad hoc

_Originally posted by @aswallace in https://github.com/AllenInstitute/biofile-finder/pull/350#discussion_r1859021220_

**Comment by SeanLeRoy** [comment]
I don't think this is necessarily worth an issue - unlikely we would circle back to this without a motivator like a bug or new feature
